composer.json

```diff
@@ -9,9 +9,9 @@
         }
     ],
     "require": {
-        "php": "^7.3|^8.0",
-        "symfony/console" : "^2.1|^3.0|^4.0|^5.0",
-        "phpunit/php-timer": "^4.0|^5.0"
+        "php": ">=8.0",
```

Comment:
Is PHP >= 8.0 required and if so, for what?
Ideally we keep the version constraint as low as possible.
If this stays then at least the workflow config needs to exclude PHP 7.4, otherwise it would always fail due to this constraint.

Reply:
I updated this to 7.4 [here](https://github.com/researchgate/phpnsc/commit/ffe027bdc35dba6229429776005a268ace36117f) but for some reason it does not show up in the PR

Reply:
Now it does. 👍